packages/interface/src/components/home/home.module.scss.d.ts

```diff
@@ -1,6 +1,7 @@
 /** This is an autogenerated file, please do not change this file directly. */
 declare namespace HomeModuleScssModule {
   export interface IHomeModuleScss {
+    darkGray: string;
```

Comment:
Oh interesting, it seems adding an export to the colors means it gets added as part of the exports of every scss file. Huh, I guess that makes sense